README.md

```diff
@@ -7303,6 +7303,30 @@ try clearDBQueue.inDatabase { db in
 // Now the copy is done, and the clear-text database can be deleted.
 ```
 
+**Advanced configuration options for SQLCipher**
```

Comment:
Please be generous in vertical whitespace. The README markdown uses `##...` prefixes for titles and subtitles.